Dockerfile

```
@@ -76,8 +73,8 @@ COPY priv priv
 RUN mix compile
 
 # Copy assets from step 1
-COPY --from=js-builder /build/priv/static priv/static
-RUN mix phx.digest
+COPY --from=js-builder /build/assets assets
```

I think we could completely remove the first step of the docker build. Running linters and formaters is a CI concern that could be handled in the GitHub Actions workflow.

```suggestion
COPY assets assets
```

---

Right, since we removed webpack, only formatting and linting dependencies remains which have no use in the release.

We still, however, need the css reset which is currently added via NPM, so we can't just remove the `npm ci` command.